src/components/Player/Pages/Records/Records.jsx

```diff
@@ -3,20 +3,36 @@ import { connect } from 'react-redux';
 import {
   getPlayerRecords,
 } from 'actions';
+import { browserHistory } from 'react-router';
 import { playerRecords } from 'reducers';
 import Table from 'components/Table';
 import Container from 'components/Container';
 import strings from 'lang';
+import dataColumns from 'components/Player/Pages/matchDataColumns';
+import ButtonGarden from 'components/ButtonGarden';
 import playerRecordsColumns from './playerRecordsColumns';
 
-const Records = ({ data, error, loading }) => (
-  <Container title={strings.heading_records} error={error} loading={loading}>
-    <Table columns={playerRecordsColumns} data={data} />
-  </Container>
-);
+const excludedColumns = ['win_rate', 'level'];
+const recordsColumns = dataColumns.filter(col => !excludedColumns.includes(col));
+
+const Records = ({ routeParams, data, error, loading, playerId }) => {
+  const selected = routeParams.subInfo || recordsColumns[0];
+  return (<div style={{ fontSize: 10 }}>
+    <ButtonGarden
+      onClick={(buttonName) => {
+        browserHistory.push(`/players/${playerId}/records/${buttonName}${window.location.search}`);
+      }}
+      buttonNames={recordsColumns}
+      selectedButton={selected}
+    />
+    <Container title={strings.heading_records} error={error} loading={loading}>
+      <Table columns={playerRecordsColumns.concat({ displayName: strings[`th_${selected}`], field: selected })} data={data} />
```

Comment:
Also, my stuns records has values with 6 decimal places. If you can add a displayFn to solve that. If you don't want to add it to everything, you can use `Math.round(x * 100) / 100` instead of the usual `toFixed` to maintain whole numbers.